service/history/queue/transfer_queue_processor_base.go

```diff
@@ -206,7 +225,10 @@ processorPumpLoop:
 					// if redispatcher still has a large number of tasks
 					// this only happens when system is under very high load
 					// we should backoff here instead of keeping submitting tasks to task processor
-					time.Sleep(loadQueueTaskThrottleRetryDelay)
+					time.Sleep(backoff.JitDuration(
```

Comment:
do you need a jitter here?

Reply:
Just following the pattern for other time intervals. I think jitter might still be necessary as when the host level task chan is full, multiple queue processor will encounter the problem at almost the same time.